main.nf

```diff
@@ -52,19 +53,20 @@
  */
 
 // Pipeline version
-version = 0.1
+version = 0.2
 
 // Reference genome index
 params.genome = 'GRCh37'
 params.index = params.genomes[ params.genome ].star
 params.gtf   = params.genomes[ params.genome ].gtf
 params.bed12 = params.genomes[ params.genome ].bed12
 
+single='testing'
```

Comment:
Can we set this to something else instead of `'testing'`? 😉  Maybe `single = null` instead..